js/game.js

```diff
@@ -200,6 +203,7 @@ function refresh(died) {
                 platFragile.push(new FragilePlatform());
             } else {
                 platSand.push(new SandPlatform());
+                platBoost.push(new BoostPlatform());
```

Comment:
From a game design perspective, I don't think this is the best way to add the blocks, since they are more helpful. On the other hand it thematically fits well as a complement to the sand blocks. We can leave this like this for now, but the overall generation method might be something to revisit in the future.